scripts/BuildLibsRelease.sh

```diff
@@ -1,113 +1,157 @@
-sudo mkdir -p /opt/local/lib
-sudo mkdir -p /opt/local/bin
+#!/bin/bash -e
```

Comment:
Reminder that this may select wildly different versions of bash depending on runtime environment because of apple's fork

Reply:
Build system is intended to always be macOS as far as I'm aware.